### ISSUE TITLE
Consistent naming convention

### ISSUE THREAD

**Issue by konsumlamm** [andrewthad/contiguous#53]
There are some inconsistent function names:

* `Mut` vs `Mutable`
  I don't really have an opinion on which is better, but I slightly prefer `Mut`, since it's shorter.
* `foldMap` vs `foldlMap`
  I don't think it makes much sense to have `l` or `r` in `foldMap`-like functions, since their point is that the order doesn't matter.

I think we should decide on a convention and then go with that, creating aliases and deprecating the old versions.

**Comment by andrewthad** [comment]
Yeah, this ended up inconsistent. If someone puts together a PR, I'd be fine with just changing everything to `Mut` and making it a major version bump. I don't think an attempt at backwards compatibility is worth it. To my understanding, there isn't a backwards compatible way to rename an associated type family (`Mutable`).

**Comment by chessai** [comment]
> Yeah, this ended up inconsistent. If someone puts together a PR, I'd be fine with just changing everything to `Mut` and making it a major version bump. I don't think an attempt at backwards compatibility is worth it. To my understanding, there isn't a backwards compatible way to rename an associated type family (`Mutable`).

Seconded

**Comment by konsumlamm** [comment]
To be clear, I actually meant the use of `Mut` in function names, do you mean changing that *and* the `Mutable` type family?

**Comment by andrewthad** [comment]
Just doing the function names has the nice advantage of being backwards compatible, so I'm fine with that option as well. I had interpreted your request as meaning everything, including the associated type family, but it would be fine to leave it as is since users do not often need to type it out.
